coalib/parsing/LineParser.py

```diff
@@ -78,6 +79,10 @@ def _parse(self, line):
                      [(section_override, key), ...], value, to_append (True if
                      append delimiter is found else False), comment
         """
+        if re.match('[^ ]#', line) is not None or\
+           re.match('#[^ ]', line) is not None:
+            logging.warning('The comment does not have whitespace' +
+                            ' before or after # in: ' + line.replace('\n', ''))
```

Comment:
"This comment" instead of "The comment".

Also maybe quotes around the comment? use `repr(line.replace('\n', ''))` so that you don't have to worry about internal quoting.